terraform/test-environments/modules/chef_automate_install/main.tf

```diff
@@ -234,9 +234,6 @@ EOF
 [global.v1]
   fqdn = "${var.alb_fqdn == "" ? element(var.instance_fqdn, count.index) : var.alb_fqdn}"
 
-[auth_z.v1.sys.service]
```

Comment:
We could optionally make this another number greater than 300, but I think we're unlikely to hit that.